docs/Documentation/Systems/Swift/running.md

```diff
@@ -324,6 +328,7 @@ Launch with the srun command:
 ```bash
 srun   ./a.out -F
 ```
+(what is this srun line used for?)
```

Comment:
I think this is explaining the batch script below, with the a.out being a placeholder for the executable. Feel free to change with something that you think is more clear

Reply:
Got you.